README.md

```diff
@@ -84,24 +102,86 @@ ngrok http 4000
 ```
 
 
+## Use Cases
 
+There are two types of actors in LaraMessaging:
 
+- Account Holder: There is only one account holder. The Account holder can see and read the messages from the anonymous users without knowing who they are.
 
+- Anonymous User: Anonymous users are able to anonymously send maessages to the account holder
+
+For MVP:
+
+- Messages cannot be deleted from prompt by user but may be automatically deleted by the system after a designated length of time. 
```

Comment:
Personally I think a message limit in terms of number of messages is better than saying messages will stay for a fixed length of time.